packages/bot-skeleton/package.json

```diff
@@ -4,6 +4,16 @@
     "description": "Visual automation for Deriv.com",
     "main": "src/app.ts",
     "private": true,
+    "jest": {
```

Comment:
this is not needed since we have the same config on jest.config.base

Reply:
Good point 👍, it really is, I double-checked and deleted